management-account/terraform/organizations-accounts-analytical-platform.tf

```diff
@@ -0,0 +1,132 @@
+locals {
+  tags_analytical_platform = merge(local.tags_business_units.platforms, {
+    application = "Analytical Platform"
+  })
+}
+
+# Analytical Platform Data Engineering
+resource "aws_organizations_account" "analytical_platform_data_engineering" {
+  name                       = "Analytical Platform Data Engineering"
+  email                      = replace(local.aws_account_email_addresses_template, "{email}", "analytical-platform-data-engineering")
+  iam_user_access_to_billing = "ALLOW"
+  parent_id                  = aws_organizations_organizational_unit.analytical_platform.id
+
+  tags = merge(local.tags_analytical_platform, {
+    environment-name = "data-engineering"
+  })
+
+  lifecycle {
+    ignore_changes = [
+      email,
+      iam_user_access_to_billing,
+      name,
+      role_name,
+    ]
+  }
+}
+
+# Analytical Platform Data Engineering Sandbox
+resource "aws_organizations_account" "analytical_platform_data_engineering_sandbox" {
+  name                       = "Analytical Platform Data Engineering Sandbox"
+  email                      = replace(local.aws_account_email_addresses_template, "{email}", "analytics-platform-dev")
```

Comment:
It's Sandbox so could be dev email on purpose, just leaving a note incase.

Reply:
Cheers for checking! This particular account has had many reincarnations, and this is the email that is currently set - it's on my todo to change it to something more meaningful (but required logging in as the AWS root IAM user for that account) 👍🏼 .